### ISSUE TITLE
Admin Color Schemes: Add rgb support for CSS admin color variables

### ISSUE THREAD

**Issue by xristos3490** [Automattic/jetpack#29303]
### Impacted plugin

Jetpack

### What

Override all CSS colors with the `---rgb` suffix.

**Comment by jimjasson** [comment]
@xristos3490 Removing this from the SomewhereWarm backlog for now.

**Comment by xristos3490** [comment]
Yes. This fix was related to the default Calypso theme color, which has now been changed to "Modern." Therefore, there is no need to handle this for WX.
